biz.aQute.bndlib.tests/test/test/ClazzTest.java

```diff
@@ -855,4 +855,15 @@ public void testInnerClass() throws Exception {
 			assertThat(clazz.isInnerClass()).isTrue();
 		}
 	}
+
+	Object anon = new Object() {};
+
+	public void testInnerAnonymousClass() throws Exception {
+		File file = IO.getFile("bin_test/test/ClazzTest$1.class");
```

Comment:
This will be a fragile test if anyone adds another anonymous class to this class. To be safe, I think you need to create a static inner class to hold the anonymous inner class for this test.

```java
public static class AnonymousClassHolder {
   Object anon = new Object() {};
}
...
File file = IO.getFile("bin_test/test/ClazzTest$AnonymousClassHolder$1.class");
```

Reply:
I can do that!